jaeger_client/thrift_gen/agent/Agent.py

```diff
@@ -209,7 +209,7 @@ def __hash__(self):
 
   def __repr__(self):
     L = ['%s=%r' % (key, value)
-      for key, value in self.__dict__.iteritems()]
+      for key, value in six.iteritems(self.__dict__)]
```

Comment:
this makes sense, but we don't edit generated files manually, there is a section in the makefile that rewrites some of the code.

Reply:
I think you use thrift and .thirft file to gen ttypes.py file,  where can i get the .thirft files?

Reply:
`.thrift` files are in `jaeger-idl`, run `git submodule update --init`

Reply:
actually `make thrift` will do everything.

Reply:
see https://github.com/jaegertracing/jaeger-client-python/blob/master/CONTRIBUTING.md#thrift